routes/routes.py

```diff
@@ -1,12 +1,51 @@
 from flask import Blueprint
+from flask import request
+from flask import render_template
 
 router = Blueprint("router", __name__)
 
 @router.route("/check")
 def check():
     return "Congratulations! Your app works. :)"
 
-@router.route("/add", methods=["POST"])
+@router.route("/add", methods=["POST","GET"])
 def add():
     # Add logic here
-    return
\ No newline at end of file
+    if request.method == "POST":
+        no1 = request.form['no1']
+        no2 = request.form['no2']
+        no3 = int(no1) + int(no2)
+        return render_template('input.html', res= str(no3), op="Add")
+    return render_template('input.html',res="",op="Add")
+
+@router.route("/sub", methods=["POST","GET"])
+def sub():
+    # Add logic here
+    if request.method == "POST":
+        no1 = request.form['no1']
+        no2 = request.form['no2']
+        no3 = int(no1) - int(no2)
+        return render_template('input.html', res= str(no3), op="Subtract")
```

Comment:
Hi, the goal of this project is to develop a REST API. Kindly read the issue - https://github.com/IEEE-VIT/SimpleMath-Flask/issues/15 and refactor your code to accept the input as mentioned